pages/api/home/home.tsx

```diff
@@ -302,43 +309,12 @@ const Home = ({
       dispatch({ field: 'prompts', value: JSON.parse(prompts) });
     }
 
-    const conversationHistory = localStorage.getItem('conversationHistory');
-    if (conversationHistory) {
```

Comment:
This if-else block is the logic that used to pick what conversation was displayed on page load. If you had a chat selected, closed that tab, eventually opened a new one, it would open again on that chat. Since changing this flow to always open a new chat, this logic is not needed anymore. I thought about making it a setting "Remember last open chat" but I wanted to match the UX on chat.openai.com anyways which doesn't remember what chat you last had open (it has chat IDs which can be implemented later).